managed/ui/src/actions/universe.js

```diff
@@ -210,9 +210,12 @@ export function resetUniverseList() {
   };
 }
 
-export function deleteUniverse(universeUUID, isForceDelete) {
+export function deleteUniverse(universeUUID, isForceDelete, isDeleteBackups) {
   const customerUUID = localStorage.getItem('customerId');
-  const deleteRequestPayload = { isForceDelete: isForceDelete };
+  const deleteRequestPayload = {
```

Comment:
```suggestion
  const deleteRequestPayload = {
      isForceDelete,
      isDeleteBackups,
    };
```
No need of colon notation you can change it to above code.

Reply:
Done.